a3fe/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.2.0"
+__version__ = "0.1.3+22.ga2a8eb5.dirty"
```

Comment:
This file gets automatically updated according to the git tag. You can list git tags with `git tag -l`. 0.2.0 should be the most recent one - if it's not, please pull changes from the main branch.

As this gets auto-updated and isn't a deliberate code change, please revert this change. Thanks!